runtime/j9vm/valhallavmi.cpp

```diff
@@ -32,8 +32,50 @@ extern "C" {
 JNIEXPORT jarray JNICALL
 JVM_CopyOfSpecialArray(JNIEnv *env, jarray orig, jint from, jint to)
 {
-	assert(!"JVM_CopyOfSpecialArray unimplemented");
-	return NULL;
+	j9object_t origObj;
+	J9Class *origClass;
+	J9Class *componentClass;
+	UDATA origLength;
+	jint len;
+	J9VMThread *currentThread = (J9VMThread *)env;
+	J9InternalVMFunctions *vmFuncs = currentThread->javaVM->internalVMFunctions;
+	vmFuncs->internalEnterVMFromJNI(currentThread);
+
+
+	if (orig == NULL) {
```

Comment:
When comparing a variable to a constant put the constant on the left as per our coding standards.